src/Components/CreateImageWizardV2/steps/TargetEnvironment/Azure/AzureTarget.tsx

```diff
@@ -0,0 +1,312 @@
+ */
+const AzureTarget = ({
+  manual,
+  setManual,
+  isErrorFetchingDetails,
+  sourceId,
+  setSourceId,
+  tenantId,
+  setTenantId,
+  subscriptionId,
+  setSubscriptionId,
+  resourceGroup,
+  setResourceGroup,
+  resourceGroups,
+}: AzureTargetPropTypes) => {
+  const [isErrorFetchingSources, setIsErrorFetchingSources] = useState(false);
+  const clearState = () => {
+    // when the user switches from manual to automatic, reset everything
+    setSourceId(0);
+    setTenantId('');
+    setResourceGroup('');
+    setSubscriptionId('');
+  };
+
+  return (
+    <>
+      <Form>
+        <Title headingLevel="h2">Target environment - Microsoft Azure</Title>
+        <TextContent>
+          <Text>
+            Upon build, Image Builder sends the image to the selected authorized
+            Azure account. The image will be uploaded to the resource group in
+            the subscription you specify.
+          </Text>
+          <Text>
+            To authorize Image Builder to push images to Microsoft Azure, the
+            account owner must configure Image Builder as an authorized
+            application for a specific tenant ID and give it the role of
+            &quot;Contributor&quot; for the resource group you want to upload
+            to. This applies even when defining target by Source selection.
+            <br />
+            <Button
+              component="a"
+              target="_blank"
+              variant="link"
+              icon={<ExternalLinkAltIcon />}
+              iconPosition="right"
+              isInline
+              href="https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow"
+            >
+              Learn more about OAuth 2.0
+            </Button>
+          </Text>
+        </TextContent>
+        <FormGroup label="Share method:" isRequired>
+          <Radio
+            name="azure-share-source"
+            id="azure-share-source"
+            label="Use an account configured from Sources."
+            description={
+              'Use a configured source to launch environments directly from the console.'
+            }
+            checked={!manual}
+            isChecked={!manual}
+            onClick={() => {
+              setManual(false);
+              clearState();
+            }}
+          />
+          <Radio
+            name="azure-share-manual"
+            id="azure-share-manual"
+            label="Manually enter the account information."
+            checked={manual}
+            isChecked={manual}
+            onClick={() => {
+              setManual(true);
+              clearState();
+            }}
+          />
+        </FormGroup>
+        {!manual && (
+          <>
+            <SourcesSelect
+              provider="azure"
+              selectedSourceId={sourceId}
+              setSelectedSourceId={setSourceId}
+              setFetchingError={setIsErrorFetchingSources}
+            />
+            <SourcesButton />
```

Comment:
Let's move it after the source select errors

original:
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/b8da82ad-b060-4289-bb5b-716f3d87096f)


current:
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/526ad85d-647d-4f45-b511-ee6975db325d)